dynawo/sources/ModelicaCompiler/Scripts_OMC_1_13_2/utils.py

```diff
@@ -19,6 +19,7 @@
 import os
 import sys
 import itertools
+#import traceback
```

Comment:
Was it done on purpose to commit the comment or just a debug feature not cleared?